exercises/triangle/triangle.py

```diff
@@ -3,5 +3,8 @@ class TriangleError(Exception):
 
 
 class Triangle(object):
-    def __init__(self):
+    def __init__(self, x, y, z):
```

Comment:
Honestly, I don't like single character names. Could you change them to something more meaningful 

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

Reply:
what do you propose? I have two suggestions, edge_1, edge_2, edge_3 or side_a, side_b, side_c